Formula/f/faust.rb

```diff
@@ -27,6 +27,10 @@ class Faust < Formula
   depends_on "libmicrohttpd"
   depends_on "libsndfile"
   depends_on "llvm@18"
+  depends_on "zstd"
+
+  uses_from_macos "ncurses"
+  uses_from_macos "zlib"
```

Comment:
No longer needed after #202964

Reply:
I was going to merge this for current bottle and rebottle in other PR. Can alternatively squash other PR and base on master branch instead.